test/Domain/LeanCode.CQRS.Tests/LoggingResponses/LoggingOperationResponsesTests.cs

```diff
@@ -0,0 +1,36 @@
+using Autofac;
+using LeanCode.CQRS.Execution;
+using Serilog;
+using Serilog.Events;
+using Xunit;
+
+namespace LeanCode.CQRS.Tests
+{
+    public class LoggingOperationResponsesTests : BaseCQRSTests
+    {
+        [Fact]
+        public async Task Correctly_log_operation_response_and_run_next()
+        {
+            Prepare();
+
+            var appCtx = new AppContext();
+            var fakeSink = new FakeSink();
+
+            var operationResponse = "Success!";
+            var nextFunctionResult = "Completed!";
+
+            var logger = new LoggerConfiguration().MinimumLevel
+                .Is(LogEventLevel.Verbose)
+                .WriteTo.Sink(fakeSink, LogEventLevel.Verbose)
+                .CreateLogger();
+
+            var element = Container.Resolve<ResponseLoggerElement<AppContext, object, object>>(new NamedParameter("logger", logger));
+
+            var result = await element.ExecuteAsync(appCtx, operationResponse, new Func<AppContext, object, Task<object>>((_, _) => Task.FromResult<object>(nextFunctionResult)));
+
+            Assert.Equal(1, fakeSink.Messages.Count());
+            Assert.Equal("Request executed with response \"Success!\"", fakeSink.Messages[0]);
```

Comment:
Don't check for the whole message, it does not matter. Check if the message contains what you expect (so the result of executing `next`).

Also, the test checks not what should be checked :P